empress/__init__.py

```diff
@@ -0,0 +1,102 @@
+"""
+Wraps empress functionalities
+"""
+import matplotlib.pyplot as plt
+from typing import List, Iterable
+from abc import ABC, abstractmethod
+from empress.xscape.CostVector import CostVector
+
+class Drawable(ABC):
+    """
+    Implements a default draw method
+    """
+
+    @abstractmethod
+    def draw_on(self, axes: plt.Axes):
+        """
+        Draw self on matplotlib Axes
+        """
+        pass
+
+    def draw(self) -> plt.Figure:
+        """
+        Draw self as matplotlib Figure.
+        """
+        # create a new figure
+        # draw on the figure
+        # return the figure
+        pass
+
+    def draw_to_file(self, fname):
+        """
+        Draw self and save it as pdf at path fname.
+        """
+        figure = self.draw()
+        figure.savefig(fname)
+
+class ReconInputWrapper:
+    def __init__(self, parasite_tree: dict, host_tree: dict, tip_mapping: dict):
+        self._parasite_tree = parasite_tree
+        self._host_tree = host_tree
+        self._tip_mapping = tip_mapping
+
+class ReconciliationWrapper(Drawable):
+    # TODO: Replace dict with Reconciliation type
```

Comment:
Note that changing the internals of `ReconciliationWrapper` should not affect front end code.